halpybot/halpyconfig.py

```diff
@@ -138,10 +148,6 @@ class Twitter(BaseModel):
 
 
 class SystemMonitoring(BaseModel):
-    enabled: bool = True
-    anope_timer: int = 300
-    message_channel: str = "#seal-bob"
```

Comment:
unused fields?

Reply:
Yes, they were already removed in develop branch but when this was originally built they were still in the master branch. 

They are the artifacts of an attempted and aborted function that sort of just sat here abandoned for the last forever before we realized they were here.